mbrl/util/mujoco.py

```diff
@@ -217,9 +220,20 @@ def __init__(self, env: gym.wrappers.TimeLimit):
         elif "mbrl.third_party.dmc2gym" in self._env.env.__class__.__module__:
             self._enter_method = self._enter_dmcontrol
             self._exit_method = self._exit_dmcontrol
+        elif "gym.envs.robotics" in self._env.env.__class__.__module__:
+            self._enter_method = self._enter_mujoco_gym_robo
+            self._exit_method = self._exit_mujoco_gym_robo
         else:
             raise RuntimeError("Tried to freeze an unsupported environment.")
 
+    def _enter_mujoco_gym_robo(self):
+        self._init_state = self._env.unwrapped.sim.get_state()
```

Comment:
This is a guess, but try with making `self._init_state` a copy of the output of `get_state()`.

Reply:
I have tried something similar. Can you check if this is working properly now?

Reply:
I have found that the problem unfortunately still remains..

Reply:
That's unfortunate. I'll take another look soon, thanks for checking.